jobs/update-payments.js

```diff
@@ -1,23 +1,16 @@
-const dbs = require('../lib/dbs')
 const env = require('../lib/env')
-const { getActiveBilling } = require('../lib/payments')
+const { getActiveBilling, getCurrentlyPrivateAndEnabledRepos } = require('../lib/payments')
 const stripe = require('stripe')(env.STRIPE_SECRET_KEY)
-const _ = require('lodash')
 
-module.exports = async ({ accountId }) => {
-  const { repositories } = await dbs()
+module.exports = async ({ accountId, repositoryId }) => {
   const billingAccount = await getActiveBilling(accountId)
-  if (!billingAccount || !billingAccount.stripeItemId) return
+    // ignore non-stripe users
+  if (!billingAccount || !billingAccount.stripeSubscriptionId) return
```

Comment:
I just checked in greenkeeper-account and both should be set to null when the paid plan is cancelled! Will change.

Reply:
we were just wondering about the change here and in the tests.

Reply:
hm I don't remember why I changed it .. you always have both. the general subscription and the Item (a single repo). The Item has a quantity within stripe.

I'll change it back

Reply:
I found why I did it! In https://github.com/greenkeeperio/greenkeeper/blob/master/jobs/stripe-event.js#L33-L40 this seems to only set the SubscriptionId to `null`  so I thought if there are some cases in the database where that was applied to then it is saver to check for the SubscriptionId instead of the ItemId
It also assigns the field `repos` to the paymentDoc which we other wise don't have or use, I guess it was copy/pasted from the old greenkeeper

Reply:
good find! can you leave a comment in the code that refers to that other part, just so it’s clear in the future? Thanks :)

Reply:
maybe then I should leave the checks for StripeSubscriptionId and also change: https://github.com/greenkeeperio/greenkeeper/blob/master/jobs/stripe-event.js#L33-L40 to also set StripeItemId to null and not assign a `repos` field

Reply:
@espy what do you think? ^